examples/v1alpha3/hyperband-example.yaml

```diff
@@ -41,7 +42,7 @@ spec:
         - ftrl
     - name: --num-epochs
       parametertype: int
```

Comment:
Is the yaml parser case sensitive? Here, `parametertype` doesn't have the `t` capitalized like elsewhere.

Reply:
Thanks for the find. I will have a look. Actually, @johnugeorge found that the e2e hyperband test is finished quickly. Not sure if it is caused by this problem.